ui/js/components/elements/tree.jsx

```diff
@@ -38,34 +38,40 @@ d3.selection.prototype.tspans2 = function(lines, lh) {
 }
 
 
-d3.wordwrap = function (line, maxCharactersPerLine, maxLines = 2) {
-	var parts = line.split(/([^a-z][a-z]*)/), //split at non-words
+d3.wordwrap = function (line = '', maxCharactersPerLine = 43, maxLines = 3) {
```

Comment:
Max characters per line updated to have a default of 43 (43 * 3 = 129, most of AWS names are 128 maximum from what I could find).